### PR TITLE
Policiy refactor. Renamed Bucketers to Dimensions

### PR DESCRIPTION
**Pull request by danielcsant**
**Description**

First step of policy refactor. Renamed bucketers to dimensions to fit the policy draft.

This implies to rename packages, maven modules, class names and example policies

**Note:** I think StreamingContextService needs a refactor. This class is responsible of too many things: input, output, parsers, ... creation and spark-streaming-context creation. One way would be to create a factory for each component (InputFactory, ParserFactory,...)

**Reviewer**
@anistal

**Test**
1 test added

**Scalastyle**
All test passed.

### CODE REVIEW

File: plugins/dimension-dateTime/src/test/scala/com/stratio/sparkta/plugin/dimension/datetime/test/DateTimeDimensionSpec.scala [hunk 1]
```diff
@@ -14,45 +14,43 @@
...
-
-import com.stratio.sparkta.plugin.bucketer.datetime.DateTimeBucketer
 import org.scalatest.{BeforeAndAfter, BeforeAndAfterAll, Matchers, WordSpecLike}
 
 /**
  * Created by ajnavarro on 9/10/14.
  */
 @RunWith(classOf[JUnitRunner])
-class DateTimeBucketerSpec extends WordSpecLike with Matchers with BeforeAndAfter with BeforeAndAfterAll {
-  var dateTimeBucketer: DateTimeBucketer = null
```
Comment (anistal) [inline plugins/dimension-dateTime/src/test/scala/com/stratio/sparkta/plugin/dimension/datetime/test/DateTimeDimensionSpec.scala]: Null is not allowed. Use val if it is possible

File: plugins/dimension-hierarchy/src/test/scala/com/stratio/sparkta/plugin/dimension/hierarchy/test/HierarchyDimensionSpec.scala [hunk 2]
```diff
@@ -29,16 +28,16 @@ import org.scalatest.{BeforeAndAfter, BeforeAndAfterAll, Matchers, WordSpecLike}
...
 @RunWith(classOf[JUnitRunner])
-class HierarchyBucketerSpec extends WordSpecLike
+class HierarchyDimensionSpec extends WordSpecLike
 with Matchers
 with BeforeAndAfter
 with BeforeAndAfterAll
 with TableDrivenPropertyChecks {
 
-  var hbs: HierarchyBucketer = null
+  var hbs: HierarchyDimension = null
```
Comment (anistal) [inline plugins/dimension-hierarchy/src/test/scala/com/stratio/sparkta/plugin/dimension/hierarchy/test/HierarchyDimensionSpec.scala]: Null is not allowed. Use val if it is possible

File: plugins/dimension-tag/src/test/scala/com/stratio/sparkta/plugin/dimension/tag/test/TagDimensionSpec.scala [hunk 3]
```diff
@@ -13,31 +13,28 @@
...
 @RunWith(classOf[JUnitRunner])
-class TagBucketerSpec extends WordSpecLike
+class TagDimensionSpec extends WordSpecLike
 with Matchers
 with BeforeAndAfter
 with BeforeAndAfterAll
 with TableDrivenPropertyChecks {
 
-  var tagBucketer: TagBucketer = null
+  var tagBucketer: TagDimension = null
```
Comment (anistal) [inline plugins/dimension-tag/src/test/scala/com/stratio/sparkta/plugin/dimension/tag/test/TagDimensionSpec.scala]: Null is not allowed. Use val if it is possible

**Comment by danielcsant** [comment]
Done! @anistal

**Pull request merged**